src/components/plots/scatterPlot.ts

```diff
@@ -16,7 +16,11 @@ export module Plots {
     constructor() {
       super();
       this.classed("scatter-plot", true);
-      this.animator(Plots.Animator.MAIN, new Animators.Base().duration(250).delay(5));
+      var animator = new Animators.Base();
+      animator.startDelay(5);
+      animator.stepDuration(250);
+      animator.maxTotalDuration(600);
```

Comment:
Yes and thus, I believe the Plot parent should hold a static variable for the maxTotalDuration for its base animator